data/coalition/coalition missions.txt

```diff
@@ -1083,7 +1083,7 @@ mission "Saryd Couple 1"
 				`	(Go back to my ship and check if it was properly refueled.)`
 					goto ship
 			`	They pause their map-gazing and introduce themselves as Inturi and Aunaris, a young couple who are looking to move out of <origin> and build a home they've always dreamed of on another world.`
-			`	"Only, in need of money for that we are," Inturi says. "Worked in a ship as a crewmember, I did, before decided to stay here and work on ship repairs and maintenance, I had."`
+			`	"Only, in need of money for that we are," Inturi says. "Worked in a ship as a crew member, I did, before decided to stay here and work on ship repairs and maintenance, I had."`
```

Comment:
```suggestion
			`	"Only, in need of money for that we are," Inturi says. "Worked in a ship as a crewmember, I did, before decided to stay here and work on ship repairs and maintenance, I had."`
```

"[crewmember](https://en.wiktionary.org/wiki/crewmember)" is actually fine as a gender neutral form of "crewman".

Reply:
It's not specifically a term in the Oxford dictionary – at least according to what Apple Dictionary's version of Oxford.

Reply:
Dictionaries describe rather than proscribe usage. Crewmember seems to be a valid form for the gender neutral version of "crewman".

https://www.thefreedictionary.com/Crewmember

Reply:
The Cambridge dictionary explicitly [writes it in two words](https://dictionary.cambridge.org/dictionary/english/crew-member?q=crewman), and it's not in the Oxford nor the Merriam-Webster dictionary. Even google translate thinks `crew member` is the right form.
![Screenshot 2024-02-04_08:45:40](https://github.com/endless-sky/endless-sky/assets/68112292/5de12ec5-6504-4cd6-8a7f-7eb68963a91f)